.github/workflows/add-labels.yml

```diff
@@ -0,0 +1,22 @@
+name: 'Add Labels'
+on:
+  issue_comment:
+    types: [created]
+
+jobs:
+  add-labels:
+    if: ${{ !github.event.issue.pull_request && (github.event.comment.body == '/help-wanted' || github.event.comment.body == '/good-first-issue' ) }}
```

Comment:
annoyingly this workflow would run every single time a comment is created in an issue if we don't check for the specific comments we want. With the checks instead of running it will skip.  The downside is that if we want to add more labels in the future we have to update this if and the script.

Reply:
We could remove this check and just let the workflow run all the time, but that seems wasteful.

Reply:
This seems like a reasonable starting point. If we decide to support many more labels in the future, we can explore alternate solutions.